.changeset/.recoverignore

```diff
@@ -0,0 +1,10 @@
+21eae416d9fa85ea6f9be55945c44ab3da2c655b
```

Comment:
these are non-published package PRs.

I need to add a flag to changeset-recover to only include PRs from publishable packages